content/questions/this-keyword/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: this keyword
+tags:
+  - this
+order: 36
+date: '2019-10-14'
+answers:
+  - 'myFunction'
+  - 'Window // correct'
```

Comment:
This is only true when running JS in the browser. Obviously there is no window in node, so this won't be the case. Can you either:
- change the question prompt to only talk about the browser or
- change the answer to not be specific to running in the browser?

Reply:
@nas5w Apologies - I've rephrased the question to specify its use as in the browser.